### PR TITLE
Adding Chromatic icon

### LINKED ISSUE #9111: Request: Chromatic
### Brand Name

Chromatic

### Website

https://www.chromatic.com/

### Popularity Metric

The Similarweb rank is `#148,884` as seen here: https://www.similarweb.com/website/chromatic.com/#overview

### Official Resources for Icon and Color

Logo: https://github.com/chromaui/learnstorybook.com/blob/67ca17500dc2a17ff9572e96fb9e44333937420a/src/components/basics/LogoChromatic.js#L9
Logo color: #fc521f

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by CodeWithShipraJain**
**Issue:** closes #9111

**Similarweb rank:** 148,884. seen here: https://www.similarweb.com/website/chromatic.com/#overview
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2253,6 +2253,11 @@
             "hex": "80B5E3",
             "source": "https://chocolatey.org/media-kit"
         },
+        {
+            "title": "Chromatic",
+            "hex": "FC521F",
+            "source": "https://www.chromatic.com/company/about"
```
Comment (LitoMore) [inline _data/simple-icons.json:2259]:
```suggestion
            "source": "https://www.chromatic.com"
```

The about page does not contain additional material other than the header logo. Let's use the homepage link instead.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (CodeWithShipraJain) [reply _data/simple-icons.json]: Updated as per your suggestion.

**Comment by CodeWithShipraJain** [comment]
@LitoMore Kindly review the suggested changes.

**Comment by LitoMore** [comment]
@CodeWithShipraJain Please hold your fire with the `update branch` button. All participants in the PR will receive an email and GitHub app notification when you pressing that button.

In fact, there is no need to keep your branch updated to develop all the time. We will merge the develop branch before merging your PR.

I appreciate your understanding.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @CodeWithShipraJain 🎉

**Pull request merged**